main/res/values/strings.xml

```diff
@@ -252,6 +253,8 @@
     <string name="info_stored_image">New image saved to:</string>
     <string name="info_storing_static_maps">Trying to store static maps</string>
     <string name="info_cache_saved">The cache has been stored locally</string>
+    <string name="info_calculator_state_saved">Current state of the calculator has been saved to waypoint.</string>
```

Comment:
It's used when one clicks the "tick" (OK) button at the top of the calculator.  It's useful to see that the state has been saved when there's an error in the formulation and clicking the button doesn't appear to do anything else otherwise.